packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -929,6 +929,13 @@ mixin WidgetInspectorService {
     );
 
     errorJson['errorsSinceReload'] = _errorsSinceReload;
+    errorJson['renderedErrorText'] = TextTreeRenderer(
+      wrapWidth: FlutterError.wrapWidth,
+      wrapWidthProperties: FlutterError.wrapWidth,
+      maxDescendentsTruncatableNode: 5,
+    )
+        .render(details.toDiagnosticsNode(style: DiagnosticsTreeStyle.error))
```

Comment:
this indentation doesn't seem like it matches the flutter style. I think it should be indented 2 and maybe it should be on the previous line.

Reply:
Did we copy this from assertions? In that file it's all on one line but exceeds 80 characters, and I got this from Cmd+Alt+L. [80 characters is just a preference](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters) so maybe we should just keep the same as there.